spec/docx/smoke/api_rgb_color_spec.rb

```diff
@@ -9,8 +9,10 @@
 
   it 'ApiRGBColor | ToJSON method' do
     docx = builder.build_and_parse('js/docx/smoke/api_rgb_color/to_json.js')
-    first_paragraph_text = '{"rgba":{"red":0,"green":0,"blue":0,"alpha":255},"color":{"rgba":{"red":255,"green":111,"blue":61,"alpha":255},"type":"srgb"},"type":"uniColor"}'
+    expected_text = '{"rgba":{"red":0,"green":0,"blue":0,"alpha":255},"color":{"rgba":{"red":255,"green":111,"blue":61,"alpha":255},"type":"srgb"},"type":"uniColor"}'
```

Comment:
I meant that you can read the text directly from the document.
```ruby
docx.elements[0].nonempty_runs.first.text
```
Or create a ruby JSON object right away 
```ruby
json = JSON.parse(docx.elements[0].nonempty_runs.first.text)
```

Reply:
fixed in new commit

Reply:
fixed in new commit